src/excmds.ts

```diff
@@ -961,6 +974,185 @@ export function jumpprev(n = 1) {
     })
 }
 
+/**
+ * Jumps to a local mark, a global mark, or the location before the last mark jump.
+ * [a-z] are local marks, [A-Z] are global marks and '`' is the location before the last mark jump.
+ * @param key the key associated with the mark
+ */
+//#content
+export async function markjump(key: string) {
+    if (key.length !== 1) {
+        throw new Error("markjump accepts only a single letter or '`'");
+    }
+    if (key === "`") {
+        return beforemarkjump()
+    }
+    if (!/[a-z]/i.exec(key)) {
+        throw new Error("markjump accepts only a single letter or '`'");
+    }
+    if (key === key.toUpperCase()) {
+        return markjumpglobal(key);
+    }
+    return markjumplocal(key);
+}
+
+/**
+ * Jumps to a local mark.
+ * @param key the key associated with the mark
+ */
+//#content
+async function markjumplocal(key: string) {
```

Comment:
I think all these functions need to be exported. I'm not sure what happens if they aren't